<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPasswordDialog</class>
 <widget class="QDialog" name="NewPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qt Bitcoin Trader</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Mt.Gox API Key and Secret</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="2" column="1">
       <widget class="QLineEdit" name="restSignLine">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>API Key:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>restKeyLine</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="clearRestKey">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="clearRestSign">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Secret:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>restSignLine</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="restKeyLine">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Your API Key and Secret will be encrypted by AES256.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="buddy">
         <cstring>restKeyLine</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Create new Password</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="1" column="2">
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPassword">
        <property name="text">
         <string>Password:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>passwordEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="passwordEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="confirmEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="confirmLabel">
        <property name="text">
         <string>Confirm:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>confirmEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="toolButton_2">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Your password must be at least 8 characters and contain letters, digits, and special characters.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="buddy">
         <cstring>passwordEdit</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="getApiKeySecretButton">
       <property name="text">
        <string>Get new API Key and Secret</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="buttonExit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="okButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>restKeyLine</tabstop>
  <tabstop>restSignLine</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>confirmEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>clearRestKey</tabstop>
  <tabstop>clearRestSign</tabstop>
  <tabstop>toolButton</tabstop>
  <tabstop>toolButton_2</tabstop>
  <tabstop>buttonExit</tabstop>
  <tabstop>getApiKeySecretButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonExit</sender>
   <signal>clicked()</signal>
   <receiver>NewPasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>NewPasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>passwordEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>confirmEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passwordEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>confirmEdit</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passwordEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewPasswordDialog</receiver>
   <slot>checkToEnableButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewPasswordDialog</receiver>
   <slot>checkToEnableButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearRestKey</sender>
   <signal>clicked()</signal>
   <receiver>restKeyLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearRestSign</sender>
   <signal>clicked()</signal>
   <receiver>restSignLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restKeyLine</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewPasswordDialog</receiver>
   <slot>checkToEnableButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>669</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restSignLine</sender>
   <signal>textChanged(QString)</signal>
   <receiver>NewPasswordDialog</receiver>
   <slot>checkToEnableButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>669</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>restKeyLine</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restKeyLine</sender>
   <signal>returnPressed()</signal>
   <receiver>restSignLine</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restSignLine</sender>
   <signal>returnPressed()</signal>
   <receiver>okButton</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>615</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>getApiKeySecretButton</sender>
   <signal>clicked()</signal>
   <receiver>NewPasswordDialog</receiver>
   <slot>getApiKeySecretButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkToEnableButton()</slot>
  <slot>getApiKeySecretButton()</slot>
 </slots>
</ui>
